### ISSUE TITLE
JAWS to read the column name in edit mode in IE browser

### ISSUE THREAD

**Issue by boyandim** [telerik/kendo-ui-core#2824]
JAWS does not read the column name in edit mode in IE browser. It only reads the cell content, but not the column name. Currently Google Chrome reads the column name as expected in edit mode. 

Please refer to the http://dojo.telerik.com/eYaka/5 example in IE browser.

### Expected/desired behavior
The column name to be read when user focuses a cell in edit mode. For example if inline edit mode is set. 

### Environment

* **Kendo UI version:** 2017.1.118
* **jQuery version:** 1.12.3
* **Browser:** [all ]

**Comment by Dimitar-Goshev** [comment]
Closing this issue as IE is not supported since R1 2023.
